qemu/tests/hugepage_mem_stress.py

```diff
@@ -35,6 +39,17 @@ def heavyload_install():
     vm.verify_alive()
     session = vm.wait_for_login()
 
+    if params.get("non_existent_point"):
+        error_context.context("This path /mnt/hugepage, doesn't mount "
+                              "hugepage.", logging.info)
+        if os.path.ismount("%s" % params.get("non_existent_point")):
+            process.run("umount %s" % params.get("non_existent_point"))
+            session.cmd('reboot & exit', timeout=1, ignore_all_errors=True)
```

Comment:
Could you please explain what this line is for?

Reply:
If this non_existent_point is already mounted. Then  need to umount it before boot the guest. Corrected.